go.mod

```diff
@@ -141,3 +141,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/klog v1.0.0 // indirect
 )
+
+replace golang.org/x/net v0.15.0 => golang.org/x/net v0.17.0
```

Comment:
Why are you doing a replace here?

Reply:
x/net is listed as an indirect dependency, that's why I did the replace. Would it be ok to directly update an indirect dep?

Reply:
fixed.